addons/editor-buttons-reverse-order/addon.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Fullscreen button on left",
+  "description": "Moves fullscreen button to the left, and green flag, pause and stop button to the right.",
```

Comment:
@WorldLanguages Help us out with a name and description.

Reply:
Name: Flipped stage control order
Description: Same

Reply:
I suggest "Reverse order of project controls" (the green flag and stop sign aren't stage controls). There should also be "as in Scratch 2.0" or something similar at the end of the description.

Reply:
I think that mxmou's idea for name is the best, but what with description?
Maybe something like that: This addon changes order of project controls. You can use this addon to recreate Scratch 2.0.
I don't think that we should add more information...

Reply:
Descriptions should mention everything an addon does to make it appear when searching for those things.

Reply:
This addon moves green flag, stop and pause button to the right, and other elements to the left. It changes order of those elements as in Scratch 2.0. We suggest to use this addon with Display stage on left side.
Something like that?